otter/test_files/ok_test.py

```diff
@@ -90,7 +90,9 @@ def run(self, global_environment):
         for i, test_case in enumerate(self.test_cases):
             passed, result = run_doctest(self.name + ' ' + str(i), test_case.body, global_environment)
             if passed:
-                result = 'Test case passed!'
+                result = '✅ Test case passed'
+            else:
+                result = '❌ Test case FAILED\n' + result
```

Comment:
I don't think the caps is necessary in `FAILED` since we've got the emoji

Reply:
I was on the edge about this too, changed to lower case now.